trovi/api/serializers.py

```diff
@@ -618,3 +630,56 @@ def to_representation(self, instance: Artifact) -> dict[str, JSON]:
     @property
     def context(self):
         return super(ArtifactPatchSerializer, self).context | {"patch": True}
+
+
+class ArtifactVersionMigrationSerializer(serializers.ModelSerializer):
+    """
+    Serializes a MigrateArtifactVersion request and response
+    """
+
+    class Meta:
+        model = ArtifactVersionMigration
+        read_only_fields = ["status", "message", "message_ratio"]
+        extra_kwargs = {"backend": {"write_only": True, "read_only": False}}
+        exclude = [
+            "id",
+            "artifact_version",
+            "source_urn",
+            "destination_urn",
+            "created_at",
+            "started_at",
+            "finished_at",
+        ]
+
+    def update(self, instance: ArtifactVersion, validated_data: dict):
+        raise NotImplementedError(f"Incorrect use of {self.__class__.__name__}")
+
+    def create(self, validated_data: dict[str, JSON]) -> ArtifactVersionMigration:
+        view = self.context["view"]
+        parent_artifact = view.kwargs.get("parent_lookup_artifact")
+        parent_version = view.kwargs.get("parent_lookup_version")
+        artifact = Artifact.objects.get(uuid=parent_artifact)
+        version = artifact.versions.get(slug=parent_version)
+
+        if version.migrations.filter(
+            status=ArtifactVersionMigration.MigrationStatus.IN_PROGRESS
+        ).exists():
+            raise MethodNotAllowed(
+                f"Artifact version {artifact.uuid}/{version.slug} "
+                f"already has a migration in progress."
+            )
+
+        migration = version.migrations.create(
+            backend=validated_data["backend"],
+            message="Submitted.",
+            source_urn=version.contents_urn,
+        )
+        artifact_version_migration_executor.submit(
+            lambda: migrate_artifact_version(migration)
+        )
```

Comment:
I think you can do it w/o the lambda, this is a common use-case for the `.submit` I think (assuming I'm looking at the right docs!)

```suggestion
        artifact_version_migration_executor.submit(
            migrate_artifact_version,
            migration
        )
```